### ISSUE TITLE
Make sure dartdoc for Flutter supports the new `mixin` syntax

### ISSUE THREAD

**Issue by yjbanov** [dart-lang/sdk#34642]
I'm filing this issue proactively. I haven't gotten far enough to verify if this already works or not. However, practice shows that something like this is extremely unlikely to work by accident. If we know that this doesn't work yet, we need to make sure someone starts working on it asap.

If this is already done, feel free to just close the issue.

@JekCharlsonYu @jcollins-g @vsmenon

**Comment by jcollins-g** [comment]
This is already implemented in https://github.com/dart-lang/dartdoc/pull/1765 and is blocked on an analyzer publish, which has some ecosystem impacts we're trying to work through.

It is tracked here: https://github.com/dart-lang/dartdoc/issues/1752 and here:  https://github.com/dart-lang/language/issues/12

**Comment by yjbanov** [comment]
Cool. I'll keep this issue open. I changed the title to reflect that this is for Flutter specifically. Whatever analyzer/ecosystem issues that need to be resolved, should be resolved before we can consider this issue as fixed.

Unless the remaining issues are non-blocking?

**Comment by jcollins-g** [comment]
I would still prefer to track this in dart-lang/dartdoc#1752 as that issue is linked to the larger tracking issue, but if having this issue helps in some way you can leave this open.

**Comment by yjbanov** [comment]
Only because we've been tracking everything blocking the roll out of the new mixins into Flutter under Dart2.1 milestone here.

**Comment by jcollins-g** [comment]
I believe so -- some support definitely exists and there are no blockers I know of.
